lib/ansible/modules/cloud/azure/azure_rm_sql_server.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sql_server
+version_added: "2.5"
+short_description: Manage an Server.
+description:
+    - Create, update and delete an instance of Server.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    name:
+        description:
+            - The name of the server.
+        required: True
+    location:
+        description:
+            - Resource location.
+        required: True
+    tags:
+        description:
+            - Resource tags.
+        required: False
+    identity:
+        description:
+            - The Azure Active Directory identity of the server.
+        required: False
+        suboptions:
+            type:
+                description:
+                    - "The identity type. Set this to 'SystemAssigned' in order to automatically create and assign an Azure Active Directory principal for th
```

Comment:
I would argue that `SystemAssigned` should be the default value if this is not set or can a SQL Server not have an identity? Is there a reason why this is a nested option under the `type` key? If not can identity just by the type value or do you see there being more options available to identity?

When quoting a value, you can enclose it with `C()`, e.g. `'SystemAssigned'` should be `C(SystemAssigned)`. You can also use `I()` when referring to option option names.